scripts/appleseedMaya/AETemplates/__init__.py

```diff
@@ -85,16 +85,13 @@ def buildBody(self, nodeName):
             self.addControl('asExposure', label='Exposure')
             self.addSeparator()
             self.addControl('asNormalize', label='Normalize')
-            self.addControl('asCastIndirectLight', label='Cast Indirect Light')
```

Comment:
Not sure why you're removing those.

Reply:
It's an arealight, using OSL emission() and has no effect.